syntaxFilters.js

```diff
@@ -64,65 +64,45 @@ function headsOnWrongSide(sTree, side, strict){
 
 }
 
-// Input Gen option to remove trees that are mirror images of trees earlier in the list
-// returns true if tree is a mirror of an eariler tree
-// returns false if tree is not a mirror image of an earlier tree
+var isMirror = true;
+// returns true if tree is a mirror of an eariler tree in list
+// returns false if tree is not a mirror image of an earlier tree in list
 function mirrorImages(sTree, sTreeList) {
+	isMirror = true;
 	var mirrorImageExists = false;
 	var index = sTreeList.indexOf(sTree);
 	var reverseTree = JSON.parse(JSON.stringify(sTree));
```

Comment:
1. This variable name confuses me every time. The tree hasn't been reversed, has it?
2. Why stringify + parse rather than using the tree directly?

Reply:
1. Yes, it hasn't been reversed, sorry about that. The reverseTree variable name is left over from when I had a function that reversed the tree first and then checked to see if it was equal. I just forgot to change it after I changed the approach.
2. This is also left over from when I was reversing the tree before comparing trees. I ran into a issue where the original tree was being modified in the reverse function, so this was my way of copying the tree. 

Sorry about these, I think I just got used to the code being this way that I didn't realize it wasn't necessary anymore.